packages/mwp-app-route-plugin/src/route.js

```diff
@@ -12,6 +12,9 @@ export const onPreResponse = {
 	 */
 	method: (request: HapiRequest, reply: HapiReply) => {
 		const response = request.response;
+
+		response.header('vary', 'User-Agent')
```

Comment:
code comment? e.g. 

```
// Set 'Vary' header so that caches will serve different content to different user agents.
// Currently, we use Fastly to rewrite the user agent string to 'desktop' or 'mobile' in order
// to minimize cache fragmentation
```

it will be useful to know the reason for this setting in 10 years when it doesn't make sense any more